### DISCUSSION TITLE
Latest restic version

### DISCUSSION THREAD

**Discussion by ADMECA** [netinvent/npbackup#49] [General]
Hi,

Since last summer restic 1.16.3 was released, possible to integrate it into npbackup ?

An option in npbackup to use external restic.exe binary would be interesting.

Thanks for your work :)

**Comment by deajan** [comment]
Hello,

I am currently working (a lot) on npbackup release 3.0.0 (https://github.com/netinvent/npbackup/tree/retention-policy) which is planned by the end of the month and will of course have newer restic binary intregrated.
I am not so keen to use non tested restic binaries, since it might break some options if output wasn't tested.
Of course, I could add an `--alternate-backend-binary` parameter.

**Comment by ADMECA** [comment]
I understand the approach, it's very interesting, I am for the integration of a new "--alternate-backend-binary" parameter allowing the use of an external binary if necessary. :)

**Reply by deajan** [reply]
Issue #50 created for this.

**Comment by ADMECA** [comment]
> Hello,
> 
> I am currently working (a lot) on npbackup release 3.0.0 (https://github.com/netinvent/npbackup/tree/retention-policy) which is planned by the end of the month and will of course have newer restic binary intregrated. I am not so keen to use non tested restic binaries, since it might break some options if output wasn't tested. Of course, I could add an `--alternate-backend-binary` parameter.

Hi, some news about the next release 3.0.0 ? I can't wait :)

**Comment by deajan** [comment]
I'm working on it, last commit is less than an hour old ;)
The core design and CLI are good, but getting the group inheritance GUI right bas been a real pain (last major blocker before beta).
Generally speaking, the GUI is horrible to code right.

I do have a professionnal deadline on this, so expect within two more weeks.

**Reply by ADMECA** [reply]
Great news, and thanks for your work. ;)
